### ISSUE TITLE
Preserve validity of the code 

### ISSUE THREAD

**Issue by ShimiSun** [hpe-design/aries#44]
- [x] No build warnings 
- [x] No build errors
- [x] No runtime errors
- [x] No editor errors

**Comment by ShimiSun** [comment]
Currently, we have multiple editor errors for not using PropTypes and unused variables.
```
Also, we have multiple runtime errors as the app launches:
 compiled successfully - ready on http://localhost:3000
Warning: Each child in a list should have a unique "key" prop. See https://fb.me/react-warning-keys for more information.
    in Unknown (at AnchorGroup/index.js:9)
    in AnchorGroup (at Layout.js:42)
    in div
    in Context.Consumer
    in Context.Consumer
    in StyledComponent
    in StyledBox
    in Box
    in Box
    in Context.Consumer
    in WithTheme(Box) (at Nav/index.js:30)
    in nav
    in Context.Consumer
    in Context.Consumer
    in StyledComponent
    in StyledBox
    in Box
    in Box
    in Context.Consumer
    in WithTheme(Box) (at Nav/index.js:11)
    in div
    in Context.Consumer
    in Context.Consumer
    in StyledComponent
    in StyledBox
    in Box
    in Box
    in Context.Consumer
    in WithTheme(Box) (at Nav/index.js:10)
    in Nav (at Layout.js:41)
    in Fragment
    in Context.Consumer (at Layout.js:28)
    in div
    in Context.Consumer
    in Context.Consumer
    in StyledComponent
    in StyledGrommet
    in Context.Provider
    in Context.Provider
    in Grommet (at Layout.js:27)
    in Layout (at pages/index.js:16)
    in Fragment
    in Index
    in App (at _document.js:16)
    in Context.Provider
    in StyleSheetManager
    in Unknown
    in Context.Provider
    in Context.Provider
    in Context.Provider
    in Context.Provider
    in AppContainer
Warning: Each child in a list should have a unique "key" prop.

Check the top-level render call using <WithTheme(Box)>. See https://fb.me/react-warning-keys for more information.
    in ContentSection (at pages/index.js:21)
    in MainContent (at pages/index.js:20)
    in div
    in Context.Consumer
    in Context.Consumer
    in StyledComponent
    in StyledBox
    in Box
    in Box
    in Context.Consumer
    in WithTheme(Box) (at Layout.js:58)
    in div
    in Context.Consumer
    in Context.Consumer
    in StyledComponent
    in StyledBox
    in Box
    in Box
    in Context.Consumer
    in WithTheme(Box) (at Layout.js:52)
    in Fragment
    in Context.Consumer (at Layout.js:28)
    in div
    in Context.Consumer
    in Context.Consumer
    in StyledComponent
    in StyledGrommet
    in Context.Provider
    in Context.Provider
    in Grommet (at Layout.js:27)
    in Layout (at pages/index.js:16)
    in Fragment
    in Index
    in App (at _document.js:16)
    in Context.Provider
    in StyleSheetManager
    in Unknown
    in Context.Provider
    in Context.Provider
    in Context.Provider
    in Context.Provider
    in AppContainer
[ event ] build page: /foundation
[ wait ]  compiling ...
[ ready ] compiled successfully - ready on http://localhost:3000
[ event ] build page: /design
[ wait ]  compiling ...
[ ready ] compiled successfully - ready on http://localhost:3000
[ event ] build page: /develop
[ wait ]  compiling ...
[ ready ] compiled successfully - ready on http://localhost:3000
[ event ] build page: /resources
[ wait ]  compiling ...
[ ready ] compiled successfully - ready on http://localhost:3000
[ event ] disposing inactive page(s): /design, /develop, /resources
^C
```

**Comment by taysea** [comment]
The warning for AnchorGroup should be gone now. Regarding PropTypes, the editor errors should be gone. I have updated them for most of the aries-core components or disabled the prop types rule (temporarily) as you have done in Tile.

Regarding the warnings for MainContent, I am working on finding how to disable this eslint rule. We disable the unique array key in Grommet. This instance is a little more complicated because when I attempted to disable the rule where the children are mapped in `MainContent` it did not have an effect.
